docs/snippets/common/storybook-story-layout-param.js.mdx

```diff
@@ -1,8 +1,11 @@
 ```js
 // Button.stories.js | Button.stories.jsx | Button.stories.ts | Button.stories.tsx
 
-export const WithLayout = Template.bind({});
-WithLayout.parameters = {
-  layout: 'centered',
+import Button from './Button';
+
+export default WithLayout = {
+  parameters: {
+    layout: 'centered',
+  },
```

Comment:
```suggestion
export default WithLayout = {
  component: Button,
  parameters: {
    layout: 'centered',
  },
```

Reply:
Thanks for catching this. But it's actually wrong. This is a story and not a component. The actual snippet should be:

```js
import Button from './Button':

export default {
  component: Button,
}

export const WithLayout = {
  parameters: {
    layout: 'centered',
  },
};
```
I'll commit it shortly.

Reply:
Fixed with the latest commit. Marking this as solved